dotnet_project/CfdCsharpProject/CfdCsharpProject.csproj

```diff
@@ -0,0 +1,43 @@
+<Project Sdk="Microsoft.NET.Sdk">

```

Comment:
Multiple definition patterns in TargetFramework could not be implemented successfully.
This will be considered in the future.
(I want to support only LTS support)